go-controller/pkg/ovn/master.go

```diff
@@ -797,42 +796,38 @@ func (oc *Controller) addNodeAnnotations(node *kapi.Node, hostSubnets []*net.IPN
 			v6Addr = ip
 		}
 	}
-	gwLPRAnnotations, err := util.CreateNodeGateRouterLRPAddrAnnotation(v4Addr, v6Addr)
-	if err != nil {
-		return fmt.Errorf("failed to marshal node %q annotation for Gateway LRP IP %v",
-			node.Name, gwLRPIPs)
-	}
 
-	nodeAnnotations, err := util.CreateNodeHostSubnetAnnotation(hostSubnets)
-	if err != nil {
-		return fmt.Errorf("failed to marshal node %q annotation for subnet %s",
-			node.Name, util.JoinIPNets(hostSubnets, ","))
-	}
+	//// Retry if it fails because of potential conflict which is transient. Return error in the
+	//// case of other errors (say temporary API server down), and it will be taken care of by the
+	//// retry mechanism.
+	resultErr := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
```

Comment:
use OvnConflictBackoff ?

Reply:
@trozet  think about the problem once more, I am wondering the need to introduce the OvnConflictBackoff variable, even what you said is correct, but I would think introducing OvnConflictBackoff is not helping in any case. The update...AnnotationWithRetry() function will only backoff and retry in case of conflict (not in case of any other error), so if there is no conflict, the retry will end. If the function actually ended up with 4 retries, it means it encounters conflict 3 times, use OvnConflictBackoff will only return failure and the pod can be only created after obj_retry logic. How that is helpful? 
so I don't see why not use the recommended backoff in case of conflict.
// DefaultBackoff is the recommended backoff for a conflict where a client
// may be attempting to make an unrelated modification to a resource under
// active management by one or more controllers.

Please let me know what do you think.

Reply:
as we talked about on slack, the problem is not with the specific object being evaluated its that it may block other objects from being processed. For pods this is a big deal, because we dont want one bad pod to stop others from getting processed. Node churn is way less often, so I'm not as worried about this case and we can leave this using the default backoff for now.